src/main/java/com/redhat/devtools/lsp4ij/LanguageServiceAccessor.java

```diff
@@ -91,6 +92,27 @@ private LanguageServiceAccessor(Project project) {
 
     private final Set<LanguageServerWrapper> startedServers = new HashSet<>();
 
+    /**
+     * Check each file open in the editor and start an LS if matching one is found and is not started yet
+     */
+    public void checkCurrentlyOpenFiles() {

```

Comment:
This method should not have a project parzmeters.

It should loop for all opened projects.

Indeed if have 2 opened ptojects and you add a new ls in the first project , the new ks will appear too inbthe cond project in lsp console